icgcget/clients/download_client.py

```diff
@@ -145,16 +154,15 @@ def parse_test_ex(ex, forbidden, not_found):
         else:
             return 0
 
-    def prepend_docker_args(self, args, mnt=None, envvars=None):
+    def prepend_docker_args(self, args, mnt=None, envvars=None, cidname=None):
```

Comment:
Putting a mutable default argument would make it behave oddly if it's called multiple times.
http://docs.python-guide.org/en/latest/writing/gotchas/